Fixer.js

```diff
@@ -1 +1,32 @@
+const json = require('./email-log.json');
 
+const fixer = function () {
+
+  function uniqueEmailCount(inputArr){
```

Comment:
You don't need this inner function, you can just set the results to `uniqueEmailCount` in your return object.